lib/ui/platform_isolate.dart

```diff
@@ -0,0 +1,169 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+part of dart.ui;
+
+/// Runs [computation] on the platform thread and returns the result.
+///
```

Comment:
If the platform doesn't support a platform thread (or it wasn't implemented) - e.g. like jason mentions linux desktop embedder - this function will throw.